Unix/configeditor/configeditor.cpp

```diff
@@ -336,16 +349,22 @@ int MI_MAIN_CALL main(int argc, const char** argv)
         size_t pos = line.find("=");
         if (line.empty() || pos == string::npos)
         {
-            cout << line << std::endl;
+        	if (opts.query.empty())
+        	{
+                cout << line << std::endl;
```

Comment:
First, fix indentation here.

Next, this change is necessary because you'd never want to echo lines for a query option, yeah? That makes sense, but if somebody made a mistake and redirected to the new omiserver.conf file, that would effectively lose it.

Not sure what's right here. In one sense, it might make sense to redirect the output anyway and let you pipe it to /dev/null if you don't want it. At least that way, if you inadvertently redirected with query, you wouldn't lose your omiserver.conf file. But on the other hand, we don't normally EVER want the output for a query, so maybe that's okay. I'm just pointing it out; if you like it this way, I'm fine with that.

I think if you keep it make it flag driven (i.e. `fDontEchoFile`). That way, if other options need it down the road, they can just set the flag and you don't have to add `||` conditions here.